src/lavinmq/config.cr

```diff
@@ -217,6 +218,10 @@ module LavinMQ
       ::Log.setup(@log_level, broadcast_backend)
       target = (path = @log_file) ? path : "stdout"
       Log.info &.emit("Logger settings", level: @log_level.to_s, target: target)
+
+      if ENV.has_key?("JOURNAL_STREAM")
+        STDERR.reopen(JournalErrorWriter.new)
+      end
```

Comment:
I think we should place this first in `lavinmq.cr` to get it in use as early as possible.